sql-plugin/pom.xml

```diff
@@ -393,6 +393,32 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
+                <artifactId>scala-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.scalatest</groupId>
+                <artifactId>scalatest-maven-plugin</artifactId>
+            </plugin>
```

Comment:
These are already specified a few lines earlier, not sure why they were added again?

Reply:
copy paste error